addons/website/views/snippets/s_faq_collapse.xml

```diff
@@ -36,6 +36,27 @@
     </section>
 </template>
 
+<template id="s_faq_collapse_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-selector=".s_faq_collapse">
+            <we-select string="Card Style">
+                <we-button title="Default" data-select-class="">Default</we-button>
+                <we-button title="Light" data-select-class="s_faq_collapse_light">Light</we-button>
+                <we-button title="Boxed" data-select-class="s_faq_collapse_boxed">Boxed</we-button>
+                <we-button title="Big" data-select-class="s_faq_collapse_big">Big</we-button>
+            </we-select>
+            <we-button-group string="Collapse Icon" data-apply-to=".card-header">
+                <we-button data-select-class="card-header-left"
```

Comment:
card-header-left is not used in the template, is this actually needed? Here is a symptom:
![image](https://user-images.githubusercontent.com/10338094/132682777-00f0ac35-935b-459a-b89e-186b24f4b62c.png)

EDIT: is it in fact not a bootstrap class? :grimacing: Please follow our naming convention then (name of the snippet first, so here something like s_faq_collapse_header_right) and don't create a class for the left alignment -> think of compatibility with users who already put the snippet in their page in 14.0, they don't have the class but the editor icon should still be marked as selected (it costs nothing to support that here).